TESTS/usb_device/basic/README.md

```diff
@@ -0,0 +1,15 @@
+# Testing the USB device data toggle reset with a Linux host
+
+When you run the `tests-usb_device-basic` test suite on a Linux machine, please make
+sure that at least one of the following prerequisites are met:
+* using the Linux kernel ***4.17* or newer**,
+* using the ***eHCI*** USB driver instead of *xHCI*.
```

Comment:
@ARMmbed/mbed-os-test FYI

Reply:
@c1728p9 What's the technical reason for this limitation?

Are we expecting devlopers to have to load a specific OS driver to be able to test in Linux?

Reply:
> @c1728p9 What's the technical reason for this limitation?

Basically, an incomplete implementation of *xHCI* driver in kernels prior to 4.17. Please see the rest of [this readme](https://github.com/ARMmbed/mbed-os/blob/065df0f0e85468feaacaa30b7e3a6e884a35af61/TESTS/usb_device/basic/README.md). I've also included some links for reference.

Reply:
> Basically, an incomplete implementation of xHCI driver in kernels prior to 4.17. Please see the rest of this readme. I've also included some links for reference.

Damn, those are actually good technical reasons...

Thanks for having them in the readme.